vagrant/provisioning/roles/arkcase-ldap-config/templates/ldap.yaml

```diff
@@ -0,0 +1,49 @@
+    {{ ldap_user_domain_bean_name }}:
+      base: {{ ldap_base }}
+      authUserDn: {{ ldap_bind_user }}
+      authUserPassword: ENC({{ encrypted_ldap_bind_password }})
+      userSearchBase: {{ ldap_user_no_base }}
+      groupSearchBase: {{ ldap_group_no_base }}
+      ldapUrl: {{ ldap_url }}
+      userAttributes: cn,userAccountControl,sn,givenName,dn,distinguishedname,memberOf,sAMAccountName,userPrincipalName,uid,uidNumber,c,co,company,department,title,shadowMax,shadowLastChange,msDS-UserPasswordExpiryTimeComputed,modifyTimestamp,description,member
```

Comment:
Out of curiosity, which authentication class is the hydrated Spring bean using?

Reply:
AcmLdapSyncConfig holds the properties values and a bean is created at runtime with a name directoryId + "_sync"